tools/dataproc-edge-node/README.md

```diff
@@ -0,0 +1,117 @@
+# Dataproc Edge Node
+
+This repository provides scripts for creating Dataproc Edge Nodes in two steps.
+Step 1 creates a Dataproc cluster, installs a configuration script, and captures a VM Image.
+Step 2 launches a GCE VM Instance from the image created by Step 1.
+
+
+## What is an Edge Node?
+
+An Edge Node is a user-facing VM Instance configured to interact with a remote Dataproc cluster. 
+
+The relationship is similar to a reverse proxy placed in front of a web application.
+
+A Dataproc Edge Node can be used as an environment to store data, scripts, code and notebooks.
+
+Because an edge node is not an active member of the cluster and does not execute tasks or contain sensitive data, users can be granted SSH access and local privileges with minimal risk of impacting production workloads.
```

Comment:
Users can still SSH from edge node to main cluster. May be add cautious language here to avoid the impression that it is entirely safe to use it. You still want to control who gets to do this.